docs/contributing/development.md

```diff
@@ -1,62 +1,54 @@
-poetry install
+To use, run `nox`. This will typecheck and test using every installed version of
+Python on your system, skipping ones that are not installed. You can also run
+specific jobs:
+
+```console
+$ nox -s typecheck              # Typecheck only
+$ nox -s tests                  # Python tests
+$ nox -s build_docs -- --serve  # Build and serve the docs
+$ nox -s build_pkg              # Make an SDist and wheel
 ```
 
-After activating our environment and installing the library with Poetry we can run Jupyter lab and start testing the local distribution or we can use `make` to run the tests and lint the code.
-Now we can create a feature branch and push those changes to our fork!
+Nox handles everything for you, including setting up an temporary virtual
+environment for each run.
 
-### Using Poetry
+## Manual development environment setup
 
-If we want to use Poetry, first we need to [install it](https://python-poetry.org/docs/#installation). After installing Poetry we can use the same workflow we used for Conda, first we install the library locally:
+Create and activate a virtual environment with `venv`, which comes by default with
+Python, in the `.venv` directory:
 
 ```bash
-poetry install
+python -m venv .venv
+. .venv/bin/activate
 ```
 
-and now we can run the local Jupyter Lab and run the scripts etc. using Poetry:
+Install _earthaccess_ in editable mode with optional development dependencies:
 
 ```bash
-poetry run jupyter lab
+pip install --editable .[dev,test,docs]
 ```
 
-!!! note
-
-    You may need to use `poetry run make ...` to run commands in the environment.
-
-### Managing Dependencies
-
-If you need to add a new dependency, you should do the following:
-
-- Run `poetry add <package>` for a required (non-development) dependency
-- Run `poetry add --group=dev <package>` for a development dependency, such
-  as a testing or code analysis dependency
-
-Both commands add an entry to `pyproject.toml` with a version that is
-compatible with the rest of the dependencies.  However, `poetry` pins versions
-with a caret (`^`), which is not what we want.  Therefore, you must locate the
-new entry in `pyproject.toml` and change the `^` to `>=`.  (See
-[poetry-relax](https://github.com/zanieb/poetry-relax) for the reasoning behind
-this.)
-
-In addition, you must also add a corresponding entry to
-`ci/environment-mindeps.yaml`.  You'll notice in this file that required
-dependencies should be pinned exactly to the versions specified in
-`pyproject.toml` (after changing `^` to `>=` there), and that development
-dependencies should be left unpinned.
+## Managing Dependencies
 
-Finally, for _development dependencies only_, you must add an entry to
-`ci/environment-dev.yaml` with the same version constraint as in
-`pyproject.toml`.
+If you need to add a new dependency, edit `pyproject.toml` and insert the dependency in
+the correct location (either in the `dependencies` array or
+`[project.optional-dependencies]`.
```

Comment:
`uv` and `pixi` provide good `add` user experiences, without the need to do the weird replacement of `^`s.